packages/commercetools/composables/__tests__/useCart/useCart.spec.ts

```diff
@@ -73,7 +74,7 @@ describe('[commercetools-composables] useCart', () => {
     const currentCart = { id: 1, version: 1 };
     const response = await clear(context, { currentCart });
 
-    expect(response).toEqual(currentCart);
+    expect(response).toEqual('some cart');
```

Comment:
What do you think about storing `some cart` in some kind of a variable? I can see that it is an assertion in several places. Lines 77, 68, 55, 46, 42, 36, 14, 10, 9, 8, 7. Then we could do i.e:
```ts
const MOCKED_CART = 'some cart';

applyCartCoupon: jest.fn(() => ({ data: { cart: MOCKED_CART } }))
expect(response).toEqual(MOCKED_CART);
```

Reply:
@Baroshem I've done it! Re-review please??